examples/bri-3/src/bri/workgroup/workgroups/agents/workgroups.agent.ts

```diff
@@ -55,12 +57,14 @@ export class WorkgroupAgent {
     securityPolicy: string,
     privacyPolicy: string,
     participants: BpiSubject[],
+    status: WorkgroupStatus,
```

Comment:
I would rather go with a separate method that does only archiving.For example archiveWorkgroup method that triggers the workgroup.Archive command (that checks that the current status is Active before proceding to update it)